### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Alex-zzzz**

### CODE REVIEW

**Comment by Alex-zzzz** [comment]
Есть много сомнений но может первый блин не комом а экзешником)

**Review by Ivan95kos — CHANGES_REQUESTED** [review]
Please, fix all checks
Знімок екрана 2023-07-31 о 17 25 17

**Comment by Alex-zzzz** [comment]
Исправил mainApp и по мелочи в классах

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    Machine[] machine = {new Bulldozer(), new Excavator(), new Truck()};
 
+        {
+        for (int i=0; i <= machine.length; i++){
```
Comment (overpathz) [inline src/main/java/core/basesyntax/MainApp.java:7]: There will be java.lang.ArrayIndexOutOfBoundsException.
You have to change `int i = 0; i < strings.length; i++` or use forEach (https://www.w3schools.com/java/java_foreach_loop.asp)

**Review by overpathz — CHANGES_REQUESTED** [review]
Overall is good. Need to change something.

File: src/main/java/core/basesyntax/Machine.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Machine.java:5]: there should be an empty line between methods

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
 public class MainApp {
+    Machine[] machine = {new Bulldozer(), new Excavator(), new Truck()};
 
+        {
+        for (Machine i : machine)
+        {
+            i.doWork();
+            i.stopWork();
+        }
+    }
```
Comment (okuzan) [inline src/main/java/core/basesyntax/MainApp.java:12]: remove redundant pair of parentheses, use naming `machine` instead of just `i`

**Review by okuzan — CHANGES_REQUESTED** [review]
Please always make sure your build passes before requesting another mentor's review

**Comment by Alex-zzzz** [comment]
open request

**Comment by Alex-zzzz** [comment]
fixed may be all problem

**Review by Alex-zzzz — COMMENTED** [review]
maven check complete

**Pull request closed without merging**